backend/CMakeLists.txt

```
@@ -20,13 +20,25 @@ set(SCHEMA_LIST "../api/schema/add.json" "../api/schema/edit.json" "../api/schem
                 "../api/schema/users.json")
 pack_schemas("${SCHEMA_LIST}" schemas)
 
-add_library(user_management INTERFACE)
+add_library(user_management "include/um/user_management.hpp")
 add_library(user_management::user_management ALIAS user_management)
 
-target_compile_features(user_management INTERFACE cxx_std_14)
-target_include_directories(user_management INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-                                                     $<INSTALL_INTERFACE:include>)
-target_link_libraries(user_management INTERFACE schemas nlohmann_json_schema_validator::nlohmann_json_schema_validator)
+set_target_properties(user_management PROPERTIES LINKER_LANGUAGE CXX)
+target_compile_features(user_management PUBLIC cxx_std_14)
+target_include_directories(user_management PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+                                                  $<INSTALL_INTERFACE:include>)
```

Currently install is not supported

```suggestion
target_include_directories(user_management PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
```